eos-tech-support/eos-download-image

```diff
@@ -25,7 +25,11 @@ def log(*args):
 
 
 def check_call(*args):
-    log('$ ' + ' '.join(map(shlex.quote, args)))
+    if shlex.quote:
```

Comment:
Doh... fixed this. Also I now have a Linux Educacional VM so I can test as well. Cunning!